metapool/tests/test_metapool.py

```diff
@@ -1259,6 +1374,20 @@ def test_add_syndna_pool1_noconcentration_exc(self):
             add_syndna(test_df, syndna_pool_number='pool1',
                        syndna_concentration=None)
 
+    def test_generate_override_cycles_value(self):
+        for fp, exp, adapter_length in self.runinfos:
+            obs = generate_override_cycles_value(fp, adapter_length)
+            # if an assertion fails, print the path of the most recently
+            # processed file for debugging purposes.
+            print(fp)
+            self.assertEqual(obs, exp)
+
+    def test_generate_override_cycles_value_error(self):
+        for fp, msg, adapter_length, e_type in self.bad_runinfos:
+            with self.assertRaisesRegex(e_type, msg):
+                print(fp)
```

Comment:
Not sure if this print is needed.

Reply:
Same situation as above.  Perhaps @charles-cowart can weigh in?

Reply:
It will be printed all the time to stdout, but it will only be shown if the test fails, in which case the fp will be useful for debugging.

Reply:
I discussed it with Antonio and I mentioned I planned to revisit the logging library in the near future. I'll go ahead and issue a PR to remove the print statements in master immediately and you can merge the lastest master into this branch afterward or I can remove them after this branch has been merged. Whatever sounds easiest for everyone.

Reply:
I took the liberty of pushing directly to this branch, my apologies. The statements are removed! :)